src/components/AddNewBook.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books';
+import './addNewBook.css';
+
+const AddNewBook = () => {
+  const dispatch = useDispatch();
+  const options = ['Fantasy', 'Philosophy', 'ScienceFiction'];
+  const category = options.map((o) => (
+    <option key={o}>{o}</option>
+  ));
+
+  return (
+    <div className="new-book">
+      <div>
+        <h3>Add new Book</h3>
+      </div>
+      <div>
+        <form
+          onSubmit={(e) => {
+            const form = document.querySelector('FORM');
+            e.preventDefault();
+            const book = {
+              title: form.title.value,
+              category: form.category.value,
+              pages: 100,
+              author: 'NA',
+              completed: 0,
+              chapter: 0,
+            };
+
+            dispatch(addBook(book));
+          }}
+        >
+          <input type="text" name="title" placeholder="Title" required />
+          <select name="category" required>
+            {category}
+          </select>
+
+          <button type="submit" className="submitBtn">Add</button>
+        </form>
+      </div>
+    </div>
```

Comment:
- [ ] you should render a form with inputs for the `title` and the `author` of the book currently you have only the title please fix that

Reply:
- [ ] Well done implementing the requested change.